dapp/src/components/SetProject.astro

```diff
@@ -60,7 +58,14 @@ const TANSU_CONTRACT_ID = import.meta.env.PUBLIC_TANSU_CONTRACT_ID;
       button.disabled = true;
       try {
         setProjectId(project_name.value);
-        navigate("/commit");
+        const project = await getProject();
+        if (project && project.name && project.config && project.maintainers) {
+          console.log("project:", project);
```

Comment:
For now this is fine, I am not sure about best practices for logging to the console things.